### PR TITLE
Fix docs for 0.8.0

### PR DESCRIPTION
**Pull request by edenlightning**
Looked at multi GPU, learning rate, cluster.

### CODE REVIEW

File: docs/source/multi_gpu.rst [hunk 1]
```diff
@@ -48,11 +48,11 @@ This will make your code scale to any arbitrary number of GPUs or TPUs with Ligh
         z = torch.Tensor(2, 3)
         z = z.type_as(x, device=self.device)
 
-Every LightningModule knows what device it is on. You can access that reference via `self.device`.
+Tha LightningModule knows what device it is on. You can access the reference via `self.device`.
```
Comment (awaelchli) [inline docs/source/multi_gpu.rst:51]:
```suggestion
The LightningModule knows what device it is on. You can access the reference via `self.device`.
```

File: docs/source/multi_gpu.rst [hunk 2]
```diff
@@ -48,11 +48,11 @@ This will make your code scale to any arbitrary number of GPUs or TPUs with Ligh
...
 
 Remove samplers
 ^^^^^^^^^^^^^^^
-For multi-node or TPU training, in PyTorch we must use `torch.nn.DistributedSampler`. The
+In Pytorch, you must use `torch.nn.DistributedSampler` for multi-node or TPU training in PyTorch. The
```
Comment (awaelchli) [inline docs/source/multi_gpu.rst:55]:
```suggestion
In PyTorch, you must use `torch.nn.DistributedSampler` for multi-node or TPU training in PyTorch. The
```
(also PyTorch is mentioned twice here in the sentence)

File: docs/source/multi_gpu.rst [hunk 3]
```diff
@@ -77,15 +76,15 @@ when needed.
...
         return DataLoader(dataset)
 
-.. note:: If you don't want this behavior, disable it with `Trainer(replace_sampler_ddp=False)`
+.. note:: You can disable this behavior with `Trainer(replace_sampler_ddp=False)`
 
 .. note:: For iterable datasets, we don't do this automatically.
 
-Make model pickleable
+Make models pickleable
 ^^^^^^^^^^^^^^^^^^^^^
```
Comment (awaelchli) [inline docs/source/multi_gpu.rst:84]:
```suggestion
Make models pickleable
^^^^^^^^^^^^^^^^^^^^^^
```

File: docs/source/multi_gpu.rst [hunk 4]
```diff
@@ -119,13 +119,11 @@ is usually helpful.
     _pickle.PicklingError: Can't pickle [THIS IS THE THING TO FIND AND DELETE]:
     attribute lookup <lambda> on __main__ failed
 
-ie: in the stacktrace example here, there seems to be a lambda function somewhere in the user code
-which cannot be pickled.
 
-GPU device selection
+Select GPU devices
 --------------------
```
Comment (awaelchli) [inline docs/source/multi_gpu.rst:124]:
```suggestion
Select GPU devices
------------------
```

File: docs/source/multi_gpu.rst [hunk 5]
```diff
@@ -178,7 +176,7 @@ Note in particular the difference between `gpus=0`, `gpus=[0]` and `gpus="0"`.
 | "-1"          | str       | [0, 1, 2, ...]      | all available GPUs              |
 +---------------+-----------+---------------------+---------------------------------+
 
-CUDA flags
+Remove CUDA flags
 ^^^^^^^^^^
```
Comment (awaelchli) [inline docs/source/multi_gpu.rst:180]:
```suggestion
Remove CUDA flags
^^^^^^^^^^^^^^^^^
```

File: docs/source/multi_gpu.rst [hunk 6]
```diff
@@ -341,7 +341,7 @@ project module). However, we STRONGLY discourage this use because it has limitat
           packages=find_packages()
           )
 
-Then setup your project like so:
+2. Setup your project like so:
 
 .. code-block:: bash
```
Comment (awaelchli) [inline docs/source/multi_gpu.rst:346]: btw, to make it look nice in the rendered output, you can indent the code block to the same level as the enumeration. I usually put a tab space after the number, like so:
1. (tab)  text text text
(tab)     here goes  code  block

Comment (edenlightning) [reply docs/source/multi_gpu.rst:346]: thanks!

**Pull request merged**